packages/e2e-test-utils/README.md

```diff
@@ -11,3 +11,528 @@ npm install @wordpress/e2e-test-utils --save-dev
 ```
 
 <br/><br/><p align="center"><img src="https://s.w.org/style/images/codeispoetry.png?1" alt="Code is Poetry." /></p>
```

Comment:
https://github.com/WordPress/gutenberg/pull/13329/files#r256816506 is another case where using tokens instead of using section titles would be beneficial.